clients/cloud/groovy/README.adoc

```diff
@@ -116,7 +116,7 @@ Verify that the consumer received all the messages:
 
  # Run the consumer
  $ ./gradlew runApp -PmainClass="io.confluent.examples.clients.cloud.StreamsExample" \
-   	-PconfigPath="$HOME/.ccloud/config" \
+   	-PconfigPath="$HOME/examples/clients/cloud/groovy/example.config" \
```

Comment:
1. User may not put the `examples` repo in `$HOME/` so this may not always work as it is today.
2. Does it need the full path or would `-PconfigPath="./example.config"` work?